src/middlewared/middlewared/schema.py

```diff
@@ -404,15 +404,19 @@ class Cron(Dict):
 
     FIELDS = ['minute', 'hour', 'dom', 'month', 'dow']
 
-    def __init__(self, name, **kwargs):
+    def __init__(self, name, *attrs, **kwargs):
```

Comment:
We don't see to be using any of these changes in this file, do we need to make those?